xapian-applications/omega/docs/omegascript.rst

```diff
@@ -337,6 +337,19 @@ $map{LIST,STUFF)
 	versions). If the tabs are a problem, use $list{$map{...},}
 	to get rid of them.
 
+$match{REGEX,STRING[,OPTIONS]}
+	peform a regex match using Perl-compatible regular expressions. Returns
+	true if a match is found, else it returns an empty string.
+
+	The optional OPTIONS argument is supported by Omega 1.3.4 and later.
```

Comment:
Only just noticed — this isn't true, because we're introducing `OPTIONS` as we introduce the command. Instead, just say "The optional OPTIONS argument can contain zero or more…".